docs/operator-guide/auth/platform-auth-model.md

```diff
@@ -1,12 +1,12 @@
 # Overview
 
-In KubeRocketCI, access control is implemented through various authorization methods. User and group permissions are regulated using Keycloak, which integrates with RBAC. Permissions for third-party tools are managed using [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/). This document provides an overview of the access management entities in KubeRocketCI, including Kubernetes groups, custom resources, and Keycloak realm roles. It also details their respective permissions and the tools they are applied to.
+In KubeRocketCI, access control is implemented through various authorization methods. To control access to the resources uses single point - Keycloak using groups and roles, which integrates with Kubernetes RBAC.  Permissions for third-party tools are configured by Keycloak [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/). This document provides an overview of the access management entities in KubeRocketCI and various tools, including Kubernetes roles, Keycloak realm roles and Kubernetes custom resources. It also details their respective permissions and the tools they are applied to.
```

Comment:
Kubernetes RBAC - need to discuss.  Keycloak [custom resources] - we do not have Keycloak custom resources.